cmd/root.go

```diff
@@ -121,7 +121,7 @@ need to read from or write to the Vault instance.`,
 			// encrypt tokens with AWS KMS
 			fmt.Println("Encrypting root token...")
 			encryptedToken, errE := kmsClient.Encrypt(&kms.EncryptInput{
-				KeyId: aws.String(fullKeyID(os.Args[1], os.Args[2], region)),
+				KeyId: aws.String(fullKeyID(os.Getenv("AWS_ACCOUNT_NUMBER"), os.Getenv("AWS_KMS_KEY_ID"), region)),
```

Comment:
Would be nice at some point to be able to support proper arguments in addition to environment variables.

Reply:
There's a bunch of usages that would benefit:
* VAULT_ADDR
* AWS_REGION
* AWS_ACCOUNT_NUMBER
* AWS_KMS_KEY_ID
* CHECK_INTERVAL

Reply:
It also seems that there's no handling around what happens when these environment variables are not set. 😅

Reply:
💯%

Reply:
Suggestion. We could have this in an issue and have it addressed later.

Reply:
Agreed. Will create an issue.

Reply:
Created them as separate issues so they can be resolved in separate PRs.

https://github.com/Intellection/vault-init/issues/6
https://github.com/Intellection/vault-init/issues/7

Please comment and edit as necessary.